x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/components/with_guided_onboarding_tour.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useEffect, useState } from 'react';
+import type { FunctionComponent, ReactElement } from 'react';
+import { EuiButton, EuiText, EuiTourStep } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+type TourType = 'addIntegrationButton' | 'integrationsList';
+const getTourConfig = (packageKey: string, tourType: TourType) => {
+  if (packageKey.startsWith('endpoint') && tourType === 'addIntegrationButton') {
+    return {
+      title: i18n.translate('xpack.fleet.guidedOnboardingTour.endpointButton.title', {
+        defaultMessage: 'Add Elastic Defend',
+      }),
+      description: i18n.translate('xpack.fleet.guidedOnboardingTour.endpointButton.description', {
+        defaultMessage: 'In the next steps we’ll help you get setup with some sensible defaults.',
```

Comment:
```suggestion
        defaultMessage: 'In just a few steps, configure your data with our recommended defaults. You can change this later.',
```
It's a bit longer, but a bit more clear about what's happening.